### PR TITLE
Containers and Cloud Portability Blog Post

### PR DESCRIPTION
**Pull request by srirajan**
In a comment on the pull request, include the following information and check off the list.

 - [ ] Mini bio
Sriram (Sri) Rajan is a Principal Architect at Rackspace responsible for designing and architecting cloud based solutions for customers. He has more than 15 years of professional experience working with computer systems, networks, programming and security. Prior to joining Rackspace, Sri worked as a systems programmer at Texas State University in San Marcos, where he also earned his Masters degree in computer science. He studied and lived in the United States for a total of nine years before relocating to the U.K. in 2010.

 - [ ] Desired date of publishing
ASAP

 - [ ] Social media contact info
In: https://www.linkedin.com/in/rajansriram
Twitter: @sriramrajan

 - [ ] Redaction: (yes|no)
yes

 - [ ] Comments: (yes|no)
yes

@catlook Please review.

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/build-13847a0e06/)

**Comment by catlook** [comment]
reviewing!!

File: _posts/2018-09-28-containers-and-cloud-portability.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  - Azure
+  - aws
+
+---
+
+One of the benefits of containers is the promise of portability.  The Docker mantra it to build, ship and, run. Containers also promise the ability to, with little changes, move from a developer’s laptop to a production environment, and, in the same vein, the ability to move to from a data centre to the cloud, or, to many clouds. However, adopting containers alone does not guarantee this.  At its core containers are just a better way of packaging your applications. While they ensure a degree of technical compatibility across many clouds, they don’t ensure complete portability by themselves. In this post, we will look at some of the many considerations from the portability lens.
+
+<!-- more -->
+
+- **Container Networking**
```
[the remark this answers is not in the document]
Comment (sriramrajan) [reply _posts/2018-09-28-containers-and-cloud-portability.md]: You have the wrong mail id: sriramrajan@acm.org!

[the remark this answers is not in the document]
Comment (catlook) [reply _posts/2018-09-28-containers-and-cloud-portability.md]: Sriram, I sent this through the git system...have you check those settings?

**Comment by srirajan** [comment]
Thanks @catlook. I have made the changes.

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/build-a8a4cfdd8d/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/build-10d8f1d58a/).

**Comment by srirajan** [comment]
Formatting looks good to me

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/build-10d8f1d58a/). :bow:

**Comment by catlook** [comment]
Thanks.  One thing next time, please use a branch rather than pushing to master...makes things a bit cleaner.  Otherwise, great work and keep 'em coming!!

**Comment by srirajan** [comment]
Thanks. Will do.

Sri

**Pull request merged**